boot_default/config.txt

```diff
@@ -73,3 +73,6 @@ dtparam=audio=on
 
 # for more options see http://elinux.org/RPi_config.txt
 
+[pi3]
+# for light board
+enable_uart=1
```

Comment:
Good idea, I'll add it to both.

On 2 June 2016 at 11:35, Tom <notifications@github.com> wrote:

> In boot_default/config.txt
> <https://github.com/KanoComputing/kano-settings/pull/385#discussion_r65517444>
> :
>
> > @@ -73,3 +73,6 @@ dtparam=audio=on
> >
> >  # for more options see http://elinux.org/RPi_config.txt
> >
> > +[pi3]
> > +# for light board
> > +enable_uart=1
>
> I suppose that this comment also applies to KanoComputing/kano-updater#185
> <https://github.com/KanoComputing/kano-updater/pull/185>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/KanoComputing/kano-settings/pull/385/files/bc7ee87af8dfe65f1c6a5c9b3adf262359e7ee40#r65517444>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACRlv12E3540ND8e5i3ycIyR0nGB6w_ks5qHrHegaJpZM4IsYZk>
> .
>